_layouts/category_archives.html

```diff
@@ -3,7 +3,14 @@
 ---
 <div class="home">
 
-  <h1 class="page-heading">Archive of {{ page.type }} '{{ page.title | escape }}'</h1>
+  {% capture i18n-tag %}archive-{{ page.type }}{% endcapture %}
+  {%- capture default-title -%}
+    Archive of {{ page.type }} '%1'
+  {% endcapture %}
+  {%- capture title -%}
+    {% include i18n.html value=i18n-tag default=default-title %}
+  {% endcapture %}
```

Comment:
These `capture`s can be updated similarly to get stripped values.
```suggestion
  {% capture default-title %}Archive of {{ page.type }} '%1'{% endcapture %}
  {% capture title %}{% include i18n.html value=i18n-tag default=default-title %}{% endcapture %}
```